packages/transformer/src/IModelExporter.ts

```diff
@@ -685,12 +685,12 @@ export class IModelExporter {
       return;
     }
     Logger.logTrace(loggerCategory, `exportRelationships(${baseRelClassFullName})`);
-    const sql = `SELECT ECInstanceId FROM ${baseRelClassFullName}`;
+    const sql = `SELECT ECInstanceId, ECClassId, * FROM ${baseRelClassFullName}`;
     await this.sourceDb.withPreparedStatement(sql, async (statement: ECSqlStatement): Promise<void> => {
       while (DbResult.BE_SQLITE_ROW === statement.step()) {
-        const relInstanceId: Id64String = statement.getValue(0).getId();
-        const relProps: RelationshipProps = this.sourceDb.relationships.getInstanceProps(baseRelClassFullName, relInstanceId);
-        await this.exportRelationship(relProps.classFullName, relInstanceId); // must call exportRelationship using the actual classFullName, not baseRelClassFullName
+        const relationshipId = statement.getValue(0).getId();
+        const relationshipClass = statement.getValue(1).getClassNameForClassId();
```

Comment:
thank you for fixing that, I was planning on removing that completely overkill `relationships.getInstanceProps` during other work. Resolve this at will.